apps/admin-gui/src/app/shared/components/blacklist-list/blacklist-list.component.html

```diff
@@ -19,8 +19,8 @@
           <mat-checkbox
             (change)="$event ? masterToggle() : null"
             [aria-label]="checkboxLabel()"
-            [checked]="selection.hasValue() && isAllSelected()"
-            [indeterminate]="selection.hasValue() && !isAllSelected()"
+            [checked]="selection.hasValue() && (dataSource | isAllSelected: selection.selected.length :canBeSelected)"
+            [indeterminate]="selection.hasValue() && !(dataSource | isAllSelected: selection.selected.length :canBeSelected)"
```

Comment:
Remove no existing method `canBeSelected`. This problem is in many places, so fix it in all HTML files, where this problem occurs, please.
![image](https://user-images.githubusercontent.com/70371638/188648282-f5a41d90-ae8c-4c1c-a655-f0d29ccf34ca.png)

```suggestion
            [checked]="selection.hasValue() && (dataSource | isAllSelected: selection.selected.length)"
            [indeterminate]="selection.hasValue() && !(dataSource | isAllSelected: selection.selected.length)"
```